resources/views/profile/edit.blade.php

```diff
@@ -4,14 +4,99 @@
+                                {{ __('auth/profile.current_password') }}
+                            </label>
+                            <input type="password" id="old-password"
+                                   class="py-3 px-4 block w-full border-gray-200 rounded-lg text-sm focus:border-blue-500 focus:ring-blue-500 disabled:opacity-50 disabled:pointer-events-none dark:bg-neutral-900 dark:border-neutral-700 dark:text-neutral-400 dark:placeholder-neutral-500 dark:focus:ring-neutral-600">
+                        </div>
+                        <div>
+                            <label for="new-password" class="block text-sm mb-2 dark:text-white">
+                                {{ __('auth/profile.new_password') }}
+                            </label>
+                            <input type="password" id="new-password"
+                                   class="py-3 px-4 block w-full border-gray-200 rounded-lg text-sm focus:border-blue-500 focus:ring-blue-500 disabled:opacity-50 disabled:pointer-events-none dark:bg-neutral-900 dark:border-neutral-700 dark:text-neutral-400 dark:placeholder-neutral-500 dark:focus:ring-neutral-600">
+                        </div>
+                        <div>
+                            <label for="repeat-password"
+                                   class="block text-sm mb-2 dark:text-white">
+                                {{ __('auth/profile.repeat_password') }}
+                            </label>
+                            <input type="password" id="repeat-password"
+                                   class="py-3 px-4 block w-full border-gray-200 rounded-lg text-sm focus:border-blue-500 focus:ring-blue-500 disabled:opacity-50 disabled:pointer-events-none dark:bg-neutral-900 dark:border-neutral-700 dark:text-neutral-400 dark:placeholder-neutral-500 dark:focus:ring-neutral-600">
+                        </div>
+                        <button type="submit"
+                                class="w-full mt-3 py-3 px-4 inline-flex justify-center items-center gap-x-2 text-sm font-semibold rounded-lg border border-transparent bg-blue-600 text-white hover:bg-blue-700 disabled:opacity-50 disabled:pointer-events-none">
+                            {{ __('auth/profile.edit_password') }}
+                        </button>
+                    </div>
+                </form>
+            </div>
+            <div class="w-full md:w-2/3">
+                <p class="dark:text-white">
+                    <strong>{{ __('auth/profile.recent_orders') }}</strong>
+                </p>
+                <a href="#">
+                    <div class="dark:text-white flex flex-row">
+                        {{ __('auth/profile.all_orders') }}&nbsp;
+                        <div class="text-blue-600 hover:underline dark:text-blue-400 dark:hover:text-blue-300">
+                            {{ __('auth/profile.here') }}
+                        </div>
+                    </div>
+                </a>
```

Comment:
Alleen de 'hier' heeft het uiterlijk van een link en de rest niet. Zelf was ik er ff verward door. Misschien een idee om alles het 'link' uiterlijk te geven?

Reply:
Ja, ik was dus aan het twijfelen, want die hele tekst een link vind ik niet zo mooi..

Reply:
Ik dacht ik doe het zo, zodat mensen met een screenreader het goed kunnen begrijpen en het voor de rest er normaal uit ziet met 'hier' in het blauw en met link styling